### ISSUE TITLE
Build with libevent

### ISSUE THREAD

**Issue by poneli** [klutchell/unbound-docker#54]
Hi,

Would it be possible to get libevent support to be able to activate that as mentioned on https://nlnetlabs.nl/documentation/unbound/howto-optimise/

If i try to enable it I recieve:
unbound[1:0] warning: too many file descriptors requested. The builtinmini-event cannot handle more than 1024. Config for less fds or compile with libevent
unbound[1:0] warning: continuing with less udp ports: 984

**Comment by klutchell** [comment]
Hey @poneli, thanks for the suggestion! Can you give an example of how you are testing this feature? Are you just increasing these values as per the documentation and getting that warning?
```
# with libevent
outgoing-range: 8192
num-queries-per-thread: 4096
```

**Comment by poneli** [comment]
Hi @klutchell,

Exactly, those were the values I tested with.

Br

**Comment by klutchell** [comment]
Can you give the new build a try with `klutchell/unbound:pr-55`?

**Comment by poneli** [comment]
Works great as in unbound not complaining about available file descriptors. :)

Thanks for quick reply and action @klutchell!

**Comment by poneli** [comment]
Do you know when it's available in :latest?

**Comment by klutchell** [comment]
I have merged to main, but building takes a few hours. At that point it will be available as `klutchell/unbound:main`.

In order for it to be available as `latest` I need to bump the `1.15.0` tag as well which I would rather not do. Are you okay pinning to `main` instead?

**Comment by poneli** [comment]
Sure, thanks for the information. :)
